org.freedesktop.Sdk.Extension.texlive.yml

```diff
@@ -115,29 +145,43 @@ modules:
           - type: archive
             url: https://ftp.gnu.org/gnu/libsigsegv/libsigsegv-2.13.tar.gz
             sha256: be78ee4176b05f7c75ff03298d84874db90f4b6c9d5503f0da1226b3a3c48119
+            x-checker-data:
+              type: html
+              url: https://www.gnu.org/software/libsigsegv/
+              pattern: (https://ftp.gnu.org/gnu/libsigsegv/libsigsegv-([\d\.]+).tar.gz)
     sources:
-    - type: archive
-      url: https://deb.debian.org/debian/pool/main/c/clisp/clisp_2.49.20180218+really2.49.92.orig.tar.bz2
-      sha256: bd443a94aa9b02da4c4abbcecfc04ffff1919c0a8b0e7e35649b86198cd6bb89
-    - type: shell
-      commands:
-        - cp -p /usr/share/automake-*/config.{sub,guess} src/build-aux
+      - type: archive
+        url: https://deb.debian.org/debian/pool/main/c/clisp/clisp_2.49.20210628.gitde01f0f.orig.tar.xz
+        sha256: 8b71994ae76e2557da637b0c1663daac6b49b7128c2a2bdcb5a155b55b55ca60
+        x-checker-data:
+          type: html
+          url: https://packages.debian.org/unstable/clisp
+          version-pattern: 'Package: clisp \(1:([\d\.]+[\w]*)-\d and others\)'
+          url-template: https://deb.debian.org/debian/pool/main/c/clisp/clisp_$version.orig.tar.xz
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} src/build-aux
   - name: xindy
     config-opts:
-    - --bindir=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
-    - --datarootdir=${FLATPAK_DEST}
-    - --libdir=${FLATPAK_DEST}/texmf-dist
-    - --mandir=${FLATPAK_DEST}/texmf-dist/doc/man
+      - --bindir=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
+      - --datarootdir=${FLATPAK_DEST}
+      - --libdir=${FLATPAK_DEST}/texmf-dist
+      - --mandir=${FLATPAK_DEST}/texmf-dist/doc/man
     sources:
-    - type: archive
-      url: http://mirrors.ctan.org/indexing/xindy/base/xindy-2.5.1.tar.gz
-      sha256: 2c8ee91db7217b5776b1ee5272dd259686f7ba3ec1d25c678f75a6c03fe9ba43        
+      - type: archive
+        url: http://mirrors.ctan.org/indexing/xindy/base/xindy-2.5.1.tar.gz
+        sha256: 2c8ee91db7217b5776b1ee5272dd259686f7ba3ec1d25c678f75a6c03fe9ba43
+        x-checker-data:
+          type: html
+          url: https://ftp.cc.uoc.gr/mirrors/CTAN/indexing/xindy/base/
```

Comment:
Why are you not using `http://mirrors.ctan.org/indexing/xindy/base` here?

Reply:
Oh, good catch. I was redirected to this mirror and didn't notice.